articles/active-directory/develop/quickstarts/active-directory-javascriptspa.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Azure AD v2 JS SPA Quickstart | Microsoft Docs
```

Comment:
Another general comment...using "msal".  I talk to customers and I have had people say...what is MSAL/ADAL?  What does that mean?  Looking at the page I don't see a definition to people of what that term is.  Maybe we should stay away from using that in the docs/code?  Like "msalconfig" if I was new to the libraries I might wonder...what is that naming?

Reply:
In *More Information* section, there is a description about what MSAL is. About msalconfig - we'd need to modify the code sample - I will start a thread about it.